2D Simulation/index.js

```diff
@@ -177,20 +168,21 @@ function initLetters(possibleLetters, numBlocks) {
     return lets;
 }
 
-function flipBlock(box, event, config) {
+function flipBlock(block_id, letter, color, config) {
```

Comment:
Why are letter and color parameters here? I think that the block's letter and color should be a part of the configuration passed in